apps/ui/lib/lens/full/MyUser/MyUser.tsx

```diff
@@ -292,6 +313,23 @@ export default class MyUser extends React.Component<Props, State> {
 							/>
 						</Flex>
 					</Tab>,
+					<Tab title="Channels" key="channels">
```

Comment:
I feel like this needs a different title, something that implies that you are an agent for the channel 🤔

Reply:
`Assigned Channels`?

Reply:
@joshbwlng Yup, that will do for now I think